providers/tests/edge/cli/test_edge_command.py

```diff
@@ -27,27 +27,20 @@
 from airflow.exceptions import AirflowException
 from airflow.providers.edge.cli.edge_command import (
     _EdgeWorkerCli,
-    _get_sysinfo,
     _Job,
 )
 from airflow.providers.edge.models.edge_job import EdgeJob
 from airflow.providers.edge.models.edge_worker import EdgeWorker, EdgeWorkerState
 from airflow.utils.state import TaskInstanceState
 
-from tests_common.test_utils.config import conf_vars
+from dev.tests_common.test_utils.config import conf_vars
```

Comment:
This has moved yesterday, please adjust - else tests will fail.
```suggestion
from tests_common.test_utils.config import conf_vars
```